resources/keycloak/custom-user-provider/src/test/java/com/cvmanager/auth/provider/user/pojos/OrganizationObjectTest.java

```diff
@@ -0,0 +1,133 @@
+package com.cvmanager.auth.provider.user.pojos;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import java.util.List;
+import java.util.Map;
+
+public class OrganizationObjectTest {
+
+    @Test
+    private void listFromString() {
```

Comment:
chore: [JUnit can not exectute `private` test methods](https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-classes-and-methods). From JUnit documentation:

> It is generally recommended to omit the public modifier for test classes, test methods, and lifecycle methods unless there is a technical reason for doing so – for example, when a test class is extended by a test class in another package. Another technical reason for making classes and methods public is to simplify testing on the module path when using the Java Module System.

```suggestion
    void listFromString() {
```

Reply:
Updated all tests!